provisioner/aws-manager.js

```diff
@@ -3,11 +3,23 @@
 var Promise = require('promise');
 var assert = require('assert');
 var debug = require('debug')('aws-provisioner:aws-manager');
-var objFilter = require('../lib/objFilter');
+// XXX var objFilter = require('../lib/objFilter');
 var shuffle = require('knuth-shuffle');
 var taskcluster = require('taskcluster-client');
 var series = require('./influx-series');
 
+const MAX_ITERATIONS_FOR_STATE_RESOLUTION = 20;
+
+function dateForInflux (thingy) {
+  if (typeof thingy === 'object' && thingy.getTime) {
+    // assume this is a date object
+    return thingy.getTime();
+  } else if (typeof thingy === 'string') {
+    return new Date(thingy).getTime();
```

Comment:
Oh, if you want to be clear safe, you can check the parsing with `isFinite(new Date(thingy))` if not finite the parse failed and the date object is `NaN`. Probably overkill.